build/arm.sh

```diff
@@ -45,7 +45,7 @@ if [ -n "${1}" ]; then
 	ARMSIZE=${1}
 fi
 
-ARMIMG="${IMAGESDIR}/${PRODUCT_RELEASE}-arm-${PRODUCT_ARCH}.img"
+ARMIMG="${IMAGESDIR}/${PRODUCT_RELEASE}-${PRODUCT_TARGET}-${PRODUCT_ARCH}-${PRODUCT_DEVICE}.img"
```

Comment:
"arm" was meant for designation of the image type like all other images do, not the target arch. If we append device that should be good enough. What do you think?

Reply:
or swap PRODUCT_ARCH for PRODUCT_TARGET

Reply:
I think swap PRODUCT_ARCH with PRODUCT_TARGET is the best option. And we should do this with kernel.txz too.

Reply:
I agree with the first one. It's also true for the kernel, but in this case we need opnsense-update to be able to have device support, which it currently hasn't. Ideally that is what we wanted to avoid with high hopes for inter-device compatible arm64 images.

Reply:
Update the name with the first one then.